great_expectations/expectations/core/expect_column_pair_values_to_be_in_set.py

```diff
@@ -74,25 +51,20 @@ class ExpectColumnPairValuesToBeInSet(ColumnPairMapExpectation):
         "requirements": [],
     }
 
-    map_metric = ("column_pair_values.in_set",)
-    domain_keys = (
-        "batch_id",
-        "table",
-        "row_condition",
-        "condition_parser",
-        "column_A",
-        "column_B",
-    )
+    map_metric = "column_pair_values.in_set"
     success_keys = ("value_pairs_set", "ignore_row_if", "mostly")
 
     default_kwarg_values = {
-        "mostly": 1,
+        "column_A": None,
+        "column_B": None,
+        "or_equal": None,
```

Comment:
I think instead of adding None as a default value, we don't want a default value (i.e. they *must* provide the argument)

Reply:
Updating (in next commit).